apps/web/src/pages/Applications/ApplicantDashboardPage/components/DashboardHeading.tsx

```diff
@@ -100,6 +106,100 @@ const DashboardHeading = ({ user }: DashboardHeadingProps) => {
         description: "Subtitle for applicant dashboard hero",
       })}
     >
+      {searchParams.get(FromIapDraftQueryKey) === "true" && (
+        <Alert.Root
+          type="info"
+          dismissible
+          onDismiss={() => {
+            searchParams.delete(FromIapDraftQueryKey);
+            setSearchParams(searchParams);
+          }}
+        >
+          <Alert.Title>
+            {intl.formatMessage({
+              defaultMessage:
+                "Your draft application for the IT Apprenticeship Program for Indigenous Peoples has been successfully saved!",
+              id: "8HdqmJ",
+              description:
+                "Title for notification that a draft IAP application was saved",
+            })}
+          </Alert.Title>
+          <p data-h2-margin-bottom="base(x0.5)">
+            {intl.formatMessage({
+              defaultMessage:
+                "You’ve now landed on your Digital Talent profile page.",
+              id: "ueLK1P",
+              description:
+                "First paragraph for applicant dashboard notification welcoming an IAP user",
+            })}
+          </p>
+          <p data-h2-margin-bottom="base(x0.5)">
+            {intl.formatMessage({
+              defaultMessage:
+                "The apprenticeship program is a part of a larger digital talent initiative in the Government of Canada that has the hopes of recruiting awesome digital talent like yourself. This profile provides you access to other digital opportunities and recruitments beyond the apprenticeship program, if you choose to apply. If you’d prefer to stick to the apprenticeship program exclusively, that’s okay too!",
+              id: "B1Pa0D",
+              description:
+                "Second paragraph for applicant dashboard notification welcoming an IAP user",
+            })}
+          </p>
+          <p>
+            {intl.formatMessage({
+              defaultMessage:
+                "When you log into the account you created, you’ll be taken to this page from now on. You can use this page to review your Indigenous Apprenticeship application and track your position in the program.",
+              id: "uSAdtr",
+              description:
+                "Third paragraph for applicant dashboard notification welcoming an IAP user",
+            })}
+          </p>
+        </Alert.Root>
+      )}
+      {searchParams.get(FromIapSuccessQueryKey) === "true" && (
+        <Alert.Root
+          type="info"
+          dismissible
+          onDismiss={() => {
+            searchParams.delete(FromIapSuccessQueryKey);
+            setSearchParams(searchParams);
+          }}
+        >
```

Comment:
Similar idea here with the `live={false}`. That is, if the team agrees these should not actually have `role="alert"` 😅